test/test_data/gnn_plot.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Melanoma intralesional IL2": [
+        "Tumor, Adipocyte or Langerhans cell, Natural killer cell, CD4+ T cell, Nerve, B cell, CD4+/CD8+ T cell, CD4+ regulatory T cell, CD8+ natural killer T cell, CD8+ regulatory T cell, CD8+ T cell, Double negative regulatory T cell, T cell/null phenotype, Natural killer T cell, CD4+ natural killer T cell",
+        "cg-gnn, graph-transformer",
+        "11, 8",
+        "horizontal",
+        "1, Non-responder",
+        "3, Responder"
+    ]
+}
```

Comment:
Can you explain the reasoning here? If it will already be JSON, it seems we can provide annotation/hints that simplify parsing (e.g. key "phenotypes" -> [ "<phenotype 1>", ... ]).

Reply:
The findings format is a list of strings for each study, and after they are inserted into the DB they should probably be regarded as unordered.

Since this usage demands more structure, why not get the uploader to insert the JSON string as a single string value?

Reply:
This way the fact that the database table records are in principle unordered will not inhibit robust parsing, and also save the trouble of having to split on commas, etc.